packages/patternfly-3/patternfly-react-extensions/less/catalog-tile.less

```diff
@@ -79,7 +80,18 @@
   .catalog-tile-pf-description {
     flex: 1;
     margin-top: 15px;
-    overflow-y: hidden;
+    overflow: hidden;
+
+    &::after {
+      background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 50%);
```

Comment:
aXe does not seem to mind this. I am guessing the screen reader will just read the text and ignore it.